tests/test_embed.py

```diff
@@ -105,6 +106,24 @@ async def test_batch_embed_semantic_invalid_batch_sizes(
         await async_client.batch_semantic_embed(request=request, batch_size=batch_size)
 
 
+@pytest.mark.system_test
```

Comment:
Did you add this decorator intentionally? I'm not sure whether we have this model in our system test setup. Could you double check?

Reply:
As far as I can tell, the test runs against the public AA API which has the required model deployed.